documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -7,24 +7,36 @@
 = Setting throughput and storage limits on brokers
 
 [role="_abstract"]
-Use the _Kafka Static Quota_ plugin to set throughput and storage limits on brokers in your Kafka cluster.
-You enable the plugin and set limits by configuring the `quotas` section of the `Kafka` resource.
-You can set a byte-rate threshold and storage quotas to put limits on the clients interacting with your brokers.
+This procedure describes how to set throughput and storage limits on brokers in your Kafka cluster.
+Enable a quota plugin and configure limits using `quotas` properties in the `Kafka` resource.
 
-NOTE: Only one quota plugin can be used in Kafka. 
-By enabling this plugin, the built-in Kafka quotas plugin is automatically disabled.
-This means that you won't see per-client quota metrics, but only aggregated quota metrics.
+* The `strimzi` type enables the _Kafka Static Quota_ plugin.
+* The `kafka` type enables the built-in Kafka plugin. 
 
-You can set byte-rate thresholds for producer and consumer bandwidth.
-The total limit is distributed across all clients accessing the broker.
-For example, you can set a byte-rate threshold of 40 MBps for producers.
-If two producers are running, they are each limited to a throughput of 20 MBps.
+Only one quota plugin can be enabled. 
+The built-in Kafka plugin is enabled by default.
+Enabling the `strimzi` plugin automatically disables the built-in plugin.
 
-Storage quotas enforce the throttling of Kafka producers based on Kafka disk storage utilization when it reaches the specified limit.
-You can specify the limit in bytes or percentage of available disk space.
-The limit applies to every disk individually.
-It prevents disks filling up too quickly and exceeding their capacity.
-Full disks can lead to issues that are hard to rectify.
+You can limit clients interacting with your brokers by setting byte-rate thresholds for producer and consumer bandwidth.
+
+* Using the `strimzi` plugin, the total limit is distributed across all clients. 
+For example, setting a 40 MBps producer byte-rate threshold limits two producers to 20 MBps each.
+* Using the `kafka` plugin, limits are applied per broker. 
+For example, setting a 20 MBps producer byte-rate threshold limits two producers to 20 MBps each.
+
+Additional `strimzi` plugin options:
+
+* Set storage quotas to throttle Kafka producers based on disk storage utilization. 
+Limits can be specified in bytes (`minAvailableBytesPerVolume`) or percentage (`minAvailableRatioPerVolume`) of available disk space, applying to each disk individually. 
```

Comment:
In this case, the limit applies across the whole cluster: clients are throttled when any broker in the cluster exceeds the configured disk threshold.